packages/terra-clinical-error-view/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "terra-clinical-error-view",
+  "main": "lib/ErrorView.js",
+  "version": "0.0.0",
+  "description": "The Terra ErrorView is a display presented with an error message and location for a button.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-clinical.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Clinical",
+    "terra-error-view",
+    "ErrorView",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
```

Comment:
How should we be setting up the peerDependencies vs dependencies in these packages we are creating? These differ from your NoDataView pull request and I am pretty certain they differ from how I set them up in LabelValueView.